src/data_structure/field2D.java

```diff
@@ -46,6 +58,41 @@ public Type getType() {
         return type;
     }
 
+    public void setColumn(fieldInfo column) { // TODO: (kam193) Maybe we should raise exception, when operation is not permitted
+        this.column = column;
+        if (this.type == Type.BLANK)
+            this.type = Type.INFOCOLUMN;
+        else if (this.type == Type.INFOROW)
+            this.type = Type.INFOCOLUMNANDROW;
+    }
+
+    public void setRow(fieldInfo row) {
+        this.row = row;
+        if (this.type == Type.BLANK)
+            this.type = Type.INFOROW;
+        else if (this.type == Type.INFOCOLUMN)
+            this.type = Type.INFOCOLUMNANDROW;
+    }
+
+    @Override
+    public String toString(){
+        if (this.type == Type.BLANK){
+            return "#";
+        }
+        else if (this.type == Type.WRITABLE){
+            return "_";
+        }
+        else if (this.type == Type.INFOCOLUMN){
+            return String.format("%d\\", this.column.getSum());
```

Comment:
Maybe use final string. It could be useful if we replace our input format to other one.